.gitignore

```diff
@@ -8,3 +8,6 @@
 /javadoc
 # local build of docs with mkdocs
 /site
+# IntelliJ project files
+*.iml
+.idea
```

Comment:
Please move the IDEA specific ignore statements to your own global GIT ignore file.

Reply:
Thx i used TortoiseGit atm. I guess these files are automatically ignored in IntelliJ. So time to check if GIT integration of IntelliJ is more suitable for me. ;)